docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1e.md

```diff
@@ -0,0 +1,56 @@
+# does_zone_meet_G3_1_1e
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a given zone meets the G3_1_1e exception "Thermal zones designed with heating-only systems in the proposed design serving storage rooms, stairwells, vestibules, electrical/mechanical rooms, and restrooms not exhausting or transferring air from mechanically cooled thermal zones in the proposed design shall use system type 9 or 10 in the baseline building design."
+
+**Inputs:**
+- **P-RMD**
+- **B-RMD**
+- **zone_id**
+
+**Returns:**  
+- **result**: an ENUM YES, NO, LIKELY_VESTIBULE or MAYBE_VESTIBULE
+ 
+**Function Call:**
+- **is_a_vestibule**
+- **is_zone_mechanically_cooled**
```

Comment:
I would create a placeholder for a function "does_zone_gets_cooled_transfer_air", and call it from here so that only that new function is affected once the schema is finalized.

Reply:
created